Kickstarter-iOS/Features/ConfirmDetails/Controllers/ConfirmDetailsViewController.swift

```diff
@@ -0,0 +1,282 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public enum ConfirmDetailsLayout {
+  enum Margin {
+    static let topBottom: CGFloat = Styles.grid(3)
+    static let leftRight: CGFloat = CheckoutConstants.PledgeView.Inset.leftRight
+  }
+}
+
+protocol ConfirmDetailsViewControllerDelegate: AnyObject {}
+
+final class ConfirmDetailsViewController: UIViewController {
+  // MARK: - Properties
+
+  public weak var delegate: ConfirmDetailsViewControllerDelegate?
+
+  private lazy var titleLabel = UILabel(frame: .zero)
+
+  /// The pledge stepper used to change the pledge amount
+  private lazy var pledgeAmountViewController = {
+    PledgeAmountViewController.instantiate()
+      |> \.delegate .~ self
+  }()
+
+  /// The shipping location shown when changing shipping locations isn't an option
+  private lazy var shippingSummaryView: PledgeShippingSummaryView = {
+    PledgeShippingSummaryView(frame: .zero)
+  }()
+
+  /// The bottom-up modal for selecting a new shipping location
+  private lazy var shippingLocationViewController = {
+    PledgeShippingLocationViewController.instantiate()
+      |> \.delegate .~ self
+  }()
+
+  private lazy var localPickupLocationView = {
+    PledgeLocalPickupView(frame: .zero)
+  }()
+
+  private lazy var inputsSectionViews = {
+    [
+      self.shippingLocationViewController.view,
+      self.shippingSummaryView,
+      self.localPickupLocationView,
+      self.pledgeAmountViewController.view
+    ]
+  }()
+
+  private lazy var keyboardDimissingTapGestureRecognizer: UITapGestureRecognizer = {
+    UITapGestureRecognizer(
+      target: self,
+      action: #selector(ConfirmDetailsViewController.dismissKeyboard)
+    )
+      |> \.cancelsTouchesInView .~ false
+  }()
+
+  private lazy var rootScrollView: UIScrollView = {
+    UIScrollView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var rootStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var rootInsetStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
```

Comment:
Setup for the sections being added in the next 2 PRs. (Pledge summary table and CTA)